module/multimedia/src/Domain/Event/MultimediaCreatedEvent.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Multimedia\Domain\Event;
+
+use Ergonode\EventSourcing\Infrastructure\DomainEventInterface;
+use Ergonode\Multimedia\Domain\Entity\MultimediaId;
+use JMS\Serializer\Annotation as JMS;
+
+/**
+ */
+class MultimediaCreatedEvent implements DomainEventInterface
+{
+    /**
+     * @var MultimediaId
+     *
+     * @JMS\Type("Ergonode\Multimedia\Domain\Entity\MultimediaId")
+     */
+    private $id;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     */
+    private $name;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     */
+    private $extension;
+
+    /**
+     * @var string|null
+     * @JMS\Type("string")
+     */
+    private $mime;
+
+    /**
+     * @var int
+     * @JMS\Type("int")
+     */
+    private $size;
+
+    /**
+     * @var string
+     * @JMS\Type("string")
+     */
+    private $crc;
+
+    /**
+     * @param MultimediaId $id
+     * @param string       $name
+     * @param string       $extension
+     * @param int          $size
+     * @param string       $crc
+     * @param string|null  $mime
+     */
+    public function __construct(
+        MultimediaId $id,
+        string $name,
+        string $extension,
+        int $size,
+        string $crc,
+        ?string $mime = null
+    ) {
+        $this->id = $id;
+        $this->name = $name;
+        $this->extension = $extension;
+        $this->mime = $mime;
+        $this->size = $size;
+        $this->crc = $crc;
```

Comment:
What kind of CRC? Maybe VO will be good solution?

Reply:
done, phpdoc